explain/setup/azure_database/03_review_settings.mdx

```diff
@@ -7,8 +7,6 @@ backlink_title: 'Automated EXPLAIN: Setup'
 
 import PgSettingsRecommendations, { getAutoExplainSettingRecommendations } from "../../../components/PgSettingsRecommendations";
 
-import imgRdsParamGroup2 from '../../../images/rds_parameter_group2.gif'
-
```

Comment:
This was wrongly imported (likely from the copy&paste), it's not used so I removed it.